configure.ac

```diff
@@ -149,5 +149,7 @@ AC_FUNC_VPRINTF
 AC_CHECK_FUNCS([ftruncate gettimeofday memset munmap select socket strtol strtoull])
 AC_CHECK_FUNCS([mmap64 posix_memalign rand_r sched_getaffinity])
 
+AC_SYS_LARGEFILE
```

Comment:
you should move this much earlier.  usually earliest is better, but maybe up around line 114 should be good enough.